search-commons/src/main/java/no/unit/nva/search2/model/ResourceParameterKey.java

```diff
@@ -36,25 +38,23 @@ public enum ResourceParameterKey implements ParameterKey {
     INVALID(STRING),
     // Parameters converted to Lucene query
     CATEGORY(STRING, "entityDescription.reference.publicationInstance.type"),
-    CONTRIBUTOR(STRING, "entityDescription.contributors.identity.id"
-                        + "|entityDescription.contributors.identity.name"),
+    CONTRIBUTOR(STRING, "entityDescription.contributors.identity.name"),
     CREATED_BEFORE(DATE, LESS_THAN, "createdDate"),
     CREATED_SINCE(DATE, GREATER_THAN_OR_EQUAL_TO, "createdDate"),
     DOI(STRING, "entityDescription.reference.doi"),
     FUNDING(STRING, "fundings.identifier"),
-    FUNDING_SOURCE(STRING, "fundings.source.identifier|fundings.source.labels.*"),
+    FUNDING_SOURCE(STRING, "fundings.source.identifier|fundings.source.labels"),
     ID(STRING, "identifier"),
-    INSTITUTION(STRING, "entityDescription.contributors.affiliation.id"
```

Comment:
no more search on id?

Reply:
I need time to investigate this, it is really strange (luciene syntax weirdness)

see examples here...

https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.identity.name:Muhammad%20Yahya)
https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.identity:Muhammad%20Yahya)
https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.identity.id:Muhammad%20Yahya)
[https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.[…]yDescription.contributors.identity.name:Muhammad%20Yahya)](https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.identity.id:Muhammad%20Yahya%20OR%20entityDescription.contributors.identity.name:Muhammad%20Yahya))
og denne blir helt vill
[https://api.dev.nva.aws.unit.no/search/resources/?query=((entityDescription.contributors[…]tyDescription.contributors.identity.id:Muhammad%20Yahya))](https://api.dev.nva.aws.unit.no/search/resources/?query=((entityDescription.contributors.identity.name:Muhammad%20Yahya)%20or%20(entityDescription.contributors.identity.id:Muhammad%20Yahya))) -> 25280
og denne enda mer!
[https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.[…]ityDescription.contributors.identity.id:Muhammad%20Yahya)](https://api.dev.nva.aws.unit.no/search/resources/?query=(entityDescription.contributors.identity.name:Muhammad%20Yahya)%20and%20(entityDescription.contributors.identity.id:Muhammad%20Yahya))  -> 79986